metadata-editor/app/controllers/EditsApi.scala

```diff
@@ -42,7 +43,8 @@ object EditsApi extends Controller with ArgoHelpers {
     // their images can be imported by drag and drop instead
     // FIXME: Creating new instances? Rubbish ಠ_ಠ.
     val usageRightsData =
-      List(PrImage(), Handout(), Screengrab(), SocialMedia(), Obituary(), Pool())
+      List(PrImage(), Handout(), Screengrab(), SocialMedia(), Obituary(), Pool(),
+           StaffPhotographer("?", "?"))
```

Comment:
Sorry I know I asked before, but is there no better way to do this? If this is really the only way, can we document?